### ISSUE TITLE
Wrong string used in posting time

### ISSUE THREAD

**Issue by daveyboond** [hit-moodle/moodle-mod_hotquestion#25]
renderer.php uses get_string('early', 'assignment',...) to get the word "early", to add to the posting time. For example: "(16 mins 25 secs early)"

This is not correct English. The word should be "ago", e.g. "(16 mins 25 secs ago)"

I think you need to add a new string to the local lang file to deal with this.

**Comment by drachels** [comment]
I disagree. In the context of posting before the question is due, "early" is the correct word to use.

**Comment by daveyboond** [comment]
Well it would be, if that were the context, but it's not. If I post a question, then return to the page 5 minutes and 25 seconds later, it will say "(5 minutes 25 secs early)". The word should be "ago".

Besides, there is no "due" time for this activity.

**Comment by drachels** [comment]
I see what you mean now. I had always just glanced at those times and made the assumption they referred to Allow Access Until, but now I agree with you. It definitely should be "ago" instead of early.
